doc/conf.py

```diff
@@ -153,12 +153,10 @@
     'bootstrap_version': "3",  # default
     'navbar_links': [
         ("Get started", "getting_started"),
-        ("Tutorials", "tutorials"),
-        ("Examples", "auto_examples/index"),
+        ("Documentation", "documentation"),
         ("API", "python_reference"),
-        ("Manual", "manual/index"),
-        ("Contribute", "contribute_to_mne"),
-        ("FAQ", "faq"),
+        ("Examples", "auto_examples/index"),
```

Comment:
I'm actually wondering if we should just go immediately to putting them under the Documentation heading, as "Advanced use cases" or so. I could add a commit that does it in an hour or two if the current revision seems reasonable.